src/app/components/Messages/PuzzleTooltip.vue

```diff
@@ -20,14 +20,14 @@
   } from 'vue-property-decorator';
   @Component
   export default class PuzzleTooltip extends Vue {
-    @Prop()
-    pid !: number;
+    @Prop({ required: true })
+    pid : number = 0;
```

Comment:
Don't give a default via `=`, use `{default: 0}`;
The current code throws an error (warning?) when the component is loaded